### ISSUE TITLE
Getting "Could not determine type of attribute" warning on GraphQL endpoint

### ISSUE THREAD

**Issue by mdrayer** [pristas-peter/wp-graphql-gutenberg#23]
On a fresh site, I get the following warning on the `/graphql` endpoint:
```
Warning: Could not determine type of attribute "figureStyle" in "core/pullquote" block type. in /var/www/html/public/wp-content/plugins/wp-graphql-gutenberg/plugin.php on line 424
```

Queries to the endpoint still go through, but it seems like this should be resolved in case anyone wants to query for the `figureStyle` attribute of a Pull Quote block.

Steps to replicate issue:
1. Spin up fresh WP site (using WP v5.3.2 for this test) 
2. Install and enable both WP GraphQL v0.6.1 and WP GraphQL Gutenberg v0.1.4
3. Go to WP GraphQL Gutenberg Admin page (`/wp/wp-admin/admin.php?page=wp-graphql-gutenberg-admin`) and click "update" button
4. Go to the GraphQL endpoint (in my case `/wp/graphql`). I'd expect to see the usual JSON payload with the "GraphQL Request must include" message, but you also get the figureStyle pull quote attribute warning.

Full result:
```
$ curl http://localhost:8080/wp/graphql

Warning: Could not determine type of attribute "figureStyle" in "core/pullquote" block type. in /var/www/html/public/wp-content/plugins/wp-graphql-gutenberg/plugin.php on line 424
{"errors":[{"message":"GraphQL Request must include at least one of those two parameters: \"query\" or \"queryId\"","category":"request"}]}
```

**Comment by bigrussthesolver** [comment]
same here.

**Comment by mrclay** [comment]
I think there's a bigger problem. On posts with this problem, the PUT to the JSON API that writes the block meta data doesn't happen.

Also, this might be useful information: https://github.com/WordPress/gutenberg/blob/6f550175b781287a70e30f97b14a72d0960313ce/packages/block-library/src/pullquote/deprecated.js#L63-L64

**Comment by dsturm** [comment]
Hey,

currently I just have a quickfix by filtering the deprecated attribute `figureStyle` out of the block_type data:

```php
add_filter(
    'option_wp_graphql_gutenberg_block_types',
    function ($block_types) {
        $index = array_search('core/pullquote', array_column($block_types, 'name'));
        if (false !== $index) {
            $block_type = $block_types[$index];
            $deprecated_attributes = array_column($block_type['deprecated'], 'attributes');
            foreach ($deprecated_attributes as $i => $deprecated_set) {
                if (\array_key_exists('figureStyle', $deprecated_set)) {
                    unset($block_type['deprecated'][$i]['attributes']['figureStyle']);
                }
            }
            $block_types[$index] = $block_type;
        }
        return $block_types;
    },
    10
);
```

**Comment by pristas-peter** [comment]
Fixed in 9bae117

Will be included in a new version (0.2.0) introducing some breaking changes. You can test it out from develop branch.
